warehouse/query-core/src/test/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTermsTest.java

```diff
@@ -614,7 +649,7 @@ public void testFailedRegexNormalizersAndNRNodes() throws ParseException {
 
         // in this case the numeric normalization fails, so we need to mark as evaluation only
```

Comment:
should the comment be updated?

Reply:
yup.  this is a case where the LcNoDiacritics succeeds but the numeric fails.  updating comment